chart/redis-cluster-operator/templates/redis.kun_distributedredisclusters_crd.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Since you are targeting Helm 3, is there any reason you are installing CRD's from here instead of as they describe: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Basically there should be a `crds` directory where they live AND helm will NOT apply any updates to them after they are installed.

Is it that this method will install and update them each time?

Reply:
Ok, Nice comment. Fixed!